src/System.Data.Common/ref/project.json

```diff
@@ -1,14 +1,51 @@
 {
   "dependencies": {
-    "System.Runtime": "4.0.0",
-    "System.Threading.Tasks": "4.0.0",
-    "System.IO": "4.0.0"
+    "Microsoft.Win32.Primitives": "4.3.0-beta-24604-02",
```

Comment:
(Also, there are several entries from the src that can actually be removed as well, e.g. CodeDom; I'd pruned the list prior to trimming out some implementation, and now those entries are no longer needed.  I'll do that as well.)